packages/cli/commands/watch.js

```diff
@@ -77,7 +77,7 @@ exports.handler = async options => {
     logger.info(
       'Passing the "--disable-initial" option is no longer necessary. Running "hs watch" no longer uploads the watched directory by default.'
     );
-  } else {
+  } else if (!initialUpload) {
```

Comment:
This is also removing the message about the watch command. Should we continue to show that message even when the initial upload flag is passed?